portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,20 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.*;
+import com.google.gson.*;
```

Comment:
https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports
Please do not use wildcard imports. Be very specific as to what you plan on using from these libraries.

i.e.
import com.google.gson.Gson;